omas/omas_plot.py

```diff
@@ -1075,7 +1089,7 @@ def TGYRO_profiles_fluxes(ods, time_index=0, fig=None, axes=None, state="None",
         if len(time) == 1:
             time_index = time_index[0]
         else:
-            return ods_time_plot(TGYRO_profiles_fluxes, time, ods, time_index, fig=fig, ax={}, **kw)
+            return ods_time_plot(profiles_fluxes, time, ods, time_index, fig=fig, ax={}, **kw)
```

Comment:
`**kw` is not right. You'll need to explicitly pass all of the arguments to profiles_fluxes.

Reply:
I see, so I am guessing the difference is that I want to call the same function but now with a different time_index i.e:

``` python
    return ods_time_plot(profiles_fluxes, ods=ods, time_index=time_index, fig=fig, 

axes=axes,plotting_label=plotting_label, show_total_density=show_total_density, plot_zeff=plot_zeff)
``` 
Is there a nicer way to do this?